### PR TITLE
Made some changes tot he markdown

### PR DESCRIPTION
**Pull request by Rafase282**
Some minor adjustments

### CODE REVIEW

File: Git-Undo-Redo-A-Commit.md [hunk 1]
```diff
@@ -1,46 +1,47 @@
-#### When to UNDO/REDO?
+# When to UNDO/REDO?
 
 You would typically want to UNDO/REDO when you commit some changes to git, and realize that the changes need to be removed/reverted. This very common in scenarios for example, when you did a temporary change to some files, and forgot to revert them, then proceeded to adding them to commit accidentally.
 
-#### The UNDO/REDO workflow:
+# The UNDO/REDO workflow:
```
Comment (abhisekp) [inline Git-Undo-Redo-A-Commit.md]: Use heading 2 here

```diff
- # The UNDO/REDO workflow:
+ ## The UNDO/REDO workflow:
```

File: Git-Undo-Redo-A-Commit.md [hunk 2]
```diff
@@ -1,46 +1,47 @@
...
 
-#### Some More tricks:
+### Some More tricks:
 
 You can go back any number of commits by using `git reset --soft HEAD~n` where you want to undo last `n` commits.
 
-###### *Attribution: This article is based on a Stack Overflow question [here](http://stackoverflow.com/questions/927358/how-do-you-undo-the-last-commit/927386#927386).*
-
-[Stack Overflow] ((http://stackoverflow.com/questions/348170/undo-git-add-before-commit/348234#348234)
+#### _Attribution: This article is based on a Stack Overflow question [here](http://stackoverflow.com/questions/927358/how-do-you-undo-the-last-commit/927386#927386) and [here](http://stackoverflow.com/questions/348170/undo-git-add-before-commit/348234#348234)._
```
Comment (abhisekp) [inline Git-Undo-Redo-A-Commit.md]:
```md
#### Attribution: 
This article is based on a Stack Overflow question [here](http://stackoverflow.com/questions/927358/how-do-you-undo-the-last-commit/927386#927386) and [here](http://stackoverflow.com/questions/348170/undo-git-add-before-commit/348234#348234).
```

----
#### Attribution: 
This article is based on a Stack Overflow question [here](http://stackoverflow.com/questions/927358/how-do-you-undo-the-last-commit/927386#927386) and [here](http://stackoverflow.com/questions/348170/undo-git-add-before-commit/348234#348234).

File: Git-Undo-Redo-A-Commit.md [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
-`git commit -m "Commit 2 - Some MORE changes to the code"`
+Assuming you did some changes and made commits like:
 
-Step 1 (UNDO-ing): Revert back the last commit <br>
-`git reset --soft HEAD~ ` <br>
-Step 2: *Do the changes.* <br>
-Step 3: Add your files to the staging area<br>
-`git add <filenames or paths>` or `git add --all` <br>
-Step 4 (REDO-ing): Do the commit. <br>
+```git
```
Comment (abhisekp) [inline Git-Undo-Redo-A-Commit.md]: Use ~~git~~ **sh**

```diff
- ```git
+ ```sh
```

----
**Supported Language List:**
:point_right: https://github.com/isagalaev/highlight.js/tree/master/src/languages

**Comment by Rafase282** [comment]
I'll probably end up just taking over the original PR so things move faster.

**Comment by abhisekp** [comment]
@Rafase282 best option :+1: 

Please use the suggestions as mentioned here.

**Pull request closed without merging**